src/qt/pivx/forms/masternodewizarddialog.ui

```diff
@@ -885,7 +885,7 @@
              </size>
             </property>
             <property name="text">
-             <string>Set Master Node’s IP and port</string>
+             <string>Set Masternode IP and port</string>
```

Comment:
would capitalize last word to `Port` (this is a title context).

Also, would change line 969 further down to read as `Enter port` as that is an input field's label.